pkg/cmd/token.go

```diff
@@ -3,6 +3,7 @@ package cmd
 import (
 	"github.com/Azure/kubelogin/pkg/token"
```

Comment:
can we revert the change in this file as .... there is no change?

Reply:
Done for next commit.